repos/fdbt-netex-output/src/netex-convertor/sharedHelpers.ts

```diff
@@ -380,7 +380,13 @@ export const getFareStructuresElements = (
     }
 
     if ('lineName' in ticket) {
-        fareStructureElements.push(getLinesElement(ticket, lineName));
+        let isReturnAndHasAdditionalService = false;
```

Comment:
This can be `const isReturnAndHasAdditionalService = isReturnTicket(ticket) && ticket.additionalServices`